packages/neutrino/index.js

```diff
@@ -0,0 +1,49 @@
+const stringify = require('javascript-stringify');
+const sort = require('deep-sort-object');
+const yargs = require('yargs');
+const Neutrino = require('./Neutrino');
+const webpack = require('./webpack');
+
+const IDENTITY = a => a;
+
+module.exports = (middleware = { use: ['.neutrinorc.js'] }, options = {}) => {
+  const neutrino = new Neutrino(options);
+  const args = yargs.parse(process.argv);
+  const mode = args.mode || 'production';
+
+  if (!process.env.NODE_ENV) {
+    process.env.NODE_ENV = mode;
+  }
+
+  neutrino.config.mode(mode);
+  neutrino.register('webpack', webpack);
+
+  if (middleware) {
+    neutrino.use(middleware);
+  }
+
+  const adapter = {
+    output(name, override = IDENTITY) {
+      if (name === 'inspect') {
```

Comment:
I think we might want to adjust the way inspect works - at the moment it means changing eg `webpack,config.js` to use `neutrino().inspect()` -- but then that doesn't take into account mode etc.

Other ways might be:
* keep a neutrino bin script for running inspect
* allow users to add `options.inspect: true` to their `.neutrinorc.js` and then the config can be output regardless of how in gets called

For me, inspect is a pretty important feature, so it's something we'll need to improve before we release - but happy to save it to a later PR if prefered (even though it's technically a regression, and one that makes it harder to review/debug PRs).

Reply:
Yeah, I'm not sure how we want to tackle this, but maybe we can use the neutrino CLI for this.

Reply:
Added `--inspect` only to the CLI for this.